webapp/portlet/src/main/webapp/vue-apps/who-is-online-app/components/ExoWhoIsOnline.vue

```diff
@@ -42,10 +43,16 @@ export default {
     },
   },
   created() {
-    this.initOnlineUsers(this.$root.onlineUsers && this.$root.onlineUsers.users || []);
-    setInterval(function () {
-      this.retrieveOnlineUsers();
-    }.bind(this), this.delay);
+    const component = this;
+    document.onreadystatechange = function () {
```

Comment:
Use lambda function to reference `this` object, see https://community.exoplatform.com/portal/dw/activity?id=364805

Reply:
done